projects/packages/my-jetpack/_inc/admin.jsx

```diff
@@ -23,6 +23,7 @@ import {
 	VideoPressInterstitial,
 } from './components/product-interstitial';
 import GoBackLink from './components/go-back-link';
+import './style.scss';
 import styles from './style.module.scss';
```

Comment:
Why we're loading two styles here?

Reply:
one for CSS modules. Another one for SCSS. Do you think we should use the same file?

Reply:
Don't think we need to split styles, we could do the same on `.module.scss`, we already are applying styles there for `#my-jetpack-container`

Reply:
good. updating...

Reply:
Reverting changes. The suggestion doesn't work. reverting changes.

Reply:
Get this done for you on [8f6a574](https://github.com/Automattic/jetpack/pull/23090/commits/8f6a574da921166f36ccfdb30a752f65c7ff822d)